templates/cluster-template.yaml

```diff
@@ -77,7 +77,7 @@ spec:
                 value: "10"
               - name: vip_retryperiod
                 value: "2"
-              image: ghcr.io/kube-vip/kube-vip:v0.5.11
+              image: ghcr.io/kube-vip/kube-vip:v0.6.4
```

Comment:
Latest version is [v0.7.1](https://github.com/kube-vip/kube-vip/releases/tag/v0.7.1)